doc/guide/Installation.ipynb

```diff
@@ -0,0 +1,138 @@
+    "\n",
+    "The installation of the packages will take some minutes. No dialogue box should appear in the meantime. If an error happens, try to solve it looking into the details description. \n",
+    "\n",
+    "Finally, set the ``climada_python-x.y.z`` folder path into the environment using the following command::\n",
+    "```   \n",
+    "      source activate climada_env\n",
+    "      conda develop /your/path/to/climada_python-x.y.z/\n",
+    "      conda deactivate\n",
+    "```\n",
+    "   You can also do so by clicking on the green right arrow in the Anaconda GUI in the *Environments* section right to the 'climada_env' environment, select 'Open Terminal' and execute the following line::\n",
+    "```\n",
+    "      conda develop /your/path/to/climada_python-x.y.z/\n",
+    "```\n",
+    "3. **Test installation**: Before leaving the *Environments* section of Anaconda, make sure that the CLIMADA environment, *climada_env* is selected. Go to the *Home* section of Anaconda and install and launch Spyder (or your preferred editor). Open the file containing all the installation tests, ``tests_install.py`` in ``climada_python-x.y.z`` folder and execute it. If the installation has been successful, an OK will appear at the end (the execution should last less than 2 min).\n",
+    "   \n",
+    "Workaround if **Spyder** installation in Anaconda fails for *climada_env*.\n",
+    "\n",
+    "In a terminal window, enter:\n",
+    "       \n",
+    "```\n",
+    "         conda activate climada_env \n",
+    "         conda install spyder-kernels \n",
+    "         python -c \"import sys; print(sys.executable)\" \n",
+    "         --> returns a path, like /Users/XXX/opt/anaconda3/envs/climada_env/bin/python\n",
+    "```\n",
+    "       \n",
+    "Start Anaconda, create a new python_env environment (just click create and enter the name, then press Create) and install latest Spyder in there (currently 4.1.5), start this Spyder, then, after Spyder has started, navigate to Preferences > Python Interpreter > Use the following interpreter and paste the path from above (e.g. /Users/XXX/opt/anaconda3/envs/CLIMADA_env/bin/python) into the text box. Start a new IPython console and run ``tests_install.py``.\n",
+    "      \n",
+    "4. **Run tutorials**: In the *Home* section of Anaconda, with *CLIMADA_env* selected, install and launch *jupyter notebook*. A browser window will show up. Navigate to your ``CLIMADA_python-x.y.z`` repository and open ``doc/tutorial/1_main_CLIMADA.ipynb``. This is the tutorial which will guide you through all CLIMADA's functionalities. Execute each code cell to see the results, you might also edit the code cells before executing. See [tutorials](https://github.com/CLIMADA-project/climada_python/tree/main/doc/tutorial) for more information.\n",
```

Comment:
The numbering here does not render correctly (I get 1. and 1. instead of 3. and 4.). Also, I think that the work around for spyder should be put as a bullet point after the 'run tutorials'.

Reply:
https://github.com/CLIMADA-project/climada_python/pull/181/commits/8e2ea600b811460cd0d43a95bd22b6e25fb79f4d